src/TypeSpec.Extension/Emitter.Csharp/src/lib/clientModelBuilder.ts

```diff
@@ -219,7 +219,18 @@ export function createModelForService(
         }
 
         var clientName = getLibraryName(sdkContext, client.type);
-        return clientName === "Models" ? "ModelsOps" : clientName;
+        if (
+            clientName === "Models" &&
+            resolveOptions(sdkContext.emitContext)["model-namespace"] !== false
```

Comment:
Do we have any case to verify the below?

model-namespace is true:
2.1 client name is "Models": give warning and rename to "ModelsOps"

Reply:
Yes. Platform-OpenAI-TypeSpec is a case.